docs/en/guides/pipeline-integrations/administration/beyond-your-first-project.md

```diff
@@ -5,4 +5,212 @@ pagename: beyond-your-first-project
 lang: en
 ---
 
-# Beyond Your First Project
+  
+# Beyond your first project  
+  
+Here, we explain where to go once you have got your first project up and running using the {% include product %} Desktop. It covers useful common questions and topics and lists useful documentation resources.    
+  
+_Please note that this document describes functionality only available if you have taken control over a Toolkit configuration. For details, see the  [{% include product %} Integrations User Guide](https://support.shotgunsoftware.com/hc/en-us/articles/115000068574-Integrations-User-Guide#Getting%20Started%20with%20Shotgun%20Desktop)._  
+  
+#### Table of Contents:  
+  
+- [Welcome to Toolkit](#welcome-to-toolkit)  
+  
+- [Basic Configuration](#basic-configuration)  
+  
+  * [Setting up Application Paths](#setting-up-application-paths)  
+  
+  * [{% include product %} Integration](#shotgun-integration)  
+  
+  * [Adding Publishes to the {% include product %} UI](#adding-publishes-to-the-shotgun-ui)  
+  
+  * [Multiple Operating Systems](#multiple-operating-systems)  
+  
+- [Next Steps](#next-steps)  
+  
+  * [The anatomy of a Toolkit Project](#the-anatomy-of-a-toolkit-project)  
+  
+      + [Command line access](#command-line-access)  
+  
+      + [Key Configuration Files](#key-configuration-files)  
+  
+      + [Core API platform](#core-api-platform)  
+  
+      + [Further reading](#further-reading)  
+  
+  * [The pipeline Toolkit Community](#the-pipeline-toolkit-community)  
+  
+  * [Our Support Team](#our-support-team)  
```

Comment:
Let's remove the table of contents entirely. What you have here actually looks pretty good, but I don't think the document is long enough to really need one explicitly built into the document like this.